media/css/firefox/whatsnew/whatsnew-lite.scss

```diff
@@ -64,41 +64,77 @@
 
 .lite-container {
     display: flex;
-    align-items: center;
+    align-items: flex-end;
 
     @media #{$mq-lg} {
-        margin-left: 80px;
+        min-height: 200px;
+        max-width: 500px;
+        h1 {
+            margin-bottom: 20px;
+        }
+    }
+
+    @media #{$mq-xl} {
+        max-width: none;
+        h1 {
+            margin-bottom: 40px;
+        }
     }
 
     img {
         margin-right: $spacing-md;
     }
+
 }
 
 .lite-body {
-    background: url('/media/img/firefox/whatsnew/lite/background-swoosh.png') no-repeat bottom, linear-gradient(#00e4ff, #3aa1ff, #0484ff);
-    color: #fff;
+    background: #42d3ec radial-gradient(50% 50% at 50% 50%, #9CF1D7 0%, rgba(255, 255, 255, 0) 100%) no-repeat top center;
+    color: $color-gray-80;
     overflow: hidden;
     position: relative;
-    padding-top: 10%;
+    @media #{$mq-lg} {
+        background-image: url('/media/img/firefox/whatsnew/lite/background.jpg');
+        background-size: 2000px auto;
+    }
+    @media #{$mq-xl} {
+        background-size: 2400px auto;
+    }
 }
 
 .lite-text {
-    @include text-display-md;
+    font-size: 1.5rem;
```

Comment:
Nit: try and stick to standard typescales https://protocol.mozilla.org/fundamentals/typography.html#code